src/canisters/frontend/ic-asset/src/asset/config.rs

```diff
@@ -937,3 +939,18 @@ mod with_tempdir {
         Ok(())
     }
 }
+
+impl std::cmp::PartialEq<Option<&AssetProperties>> for AssetConfig {
```

Comment:
Is it really necessary to write this by hand?

Reply:
I mean, does derive(PartialEq) on AssetProperties not work?

Reply:
this impl compares AssetConfig vs AssetProperties (two different structs)